addons/mail/static/src/components/emoji_picker_view/emoji_picker_view.scss

```diff
@@ -1,9 +1,4 @@
-// ------------------------------------------------------------------
-// Layout
-// ------------------------------------------------------------------
-
 .o_EmojiPickerView {
-    // Could be replaced by "d-grid" class after migration to BS5
-    display: grid;
-    grid-template-columns: repeat(9, 1fr);
+    height: Min(80vh, 340px);
+    min-height: 0;
```

Comment:
Rules seem conflicting, either the parent is flex and height is probably useless, or the parent is not flex and min-height is probably useless if there is an hard-coded height?

As parent is not flex, `min-height: 0;` is probably useless here.